libs/tree-sitter-wing/.gitattributes

```diff
@@ -0,0 +1 @@
+/src/grammar.json linguist-generated
```

Comment:
I added the pr/e2e-full label on this PR, but maybe it shouldn't matter? If that's not needed, you can remove it before you merge that in.

Reply:
Shouldn't be needed, the file already exists in the built version